modules/reissue-internal-certificates-secured-clusters.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_module-type: PROCEDURE
+[id="reissue-internal-certificates-secured-cluster_{context}"]
+= Reissuing internal certificates for Secured Clusters using init bundles
+
+Secured clusters contain the Collector, Sensor, and Admission Control components. These components use a built-in server certificate for authentication when communicating with other {product-title} components.
+
+The RHACS portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiry date.
+====
+
+.Prerequisites
+
+* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+
+[IMPORTANT]
+====
+Store this bundle securely because it contains secrets.
+You can use the same bundle on multiple secured clusters.
+====
+
+.Procedure
+
+To generate an init bundle using the {product-title-short} portal:
```

Comment:
```suggestion
To generate an `init` bundle using the {product-title-short} portal:
```
Use backticks for all non-GUI "system items", including daemon, service, or software package. [Quick markup reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)

Reply:
Hi, throughout the docs they have previously been called init bundle (no backticks). The product was originally an acquisition so that may be why it doesn't use RH/IBM style. I am reluctant to change this for this release note, since it would be inconsistent with how it is referred to elsewhere in the docs.